6_visualize/src/create_animation_frame.R

```diff
@@ -1,7 +1,7 @@
 create_animation_frame <- function(png_file, config, ...) {
 
   # open the plotting device. this should eventually be a gif function, but i don't know how that looks yet
-  png(filename=png_file, width=config$width, height=config$height, units='px', type='cairo')
+  png(filename=png_file, width=config$width, height=config$height, units='px')
```

Comment:
Could we stick a switch/ifelse statement in here to support a best-available `type` for each OS?